core/components/CreatedTasksList/CreatedTasksList.tsx

```diff
@@ -21,22 +21,18 @@ const CreatedTasksList = ({
       <TasksLayout>
         {tasks?.map((task: TaskProps) => (
           <div key={task.id}>
-            {task.status === TaskStatusType.CREATED ? (
-              !task?.editable ? (
-                <Task
-                  task={task}
-                  handleAction={(data: string) => handleAction(data, task)}
-                  handleEdit={() => handleEdit(task)}
-                />
-              ) : (
-                <TaskForm task={task} />
-              )
+            {task.status === TaskStatusType.CREATED && !task.editMode ? (
```

Comment:
add filter to map instead of checking `CREATED`, for example: 
`tasks.filter(({status}) => status === TaskStatusType.CREATED).map...`